src/components/CreateRecipe.js

```diff
@@ -0,0 +1,343 @@
+                          onChange={() => {}}
+                          inputProps={{
+                            "data-id": idx,
+                            "data-field-type": "amount"
+                          }}
+                        />
+                      </Grid>
+                      <Grid item xs={2}>
+                        <InputLabel
+                          style={{ fontSize: "12px", paddingTop: "2px" }}
+                          htmlFor="age-native-simple"
+                        >
+                          Unit
+                        </InputLabel>
+                        <Select
+                          native
+                          value={this.props.ingredients[idx].unit}
+                          name={unitId}
+                          id={unitId}
+                          style={{ marginTop: "-6.1px" }}
+                          alt="unit"
+                          label="Unit"
+                          inputProps={{
+                            "data-id": idx,
+                            "data-field-type": "unit"
+                          }}
+                        >
+                          <option value="" />
+                          <option value={"piece"}>piece</option>
+                          <option value={"ml"}>ml</option>
+                          <option value={"g"}>g</option>
+                          <option value={"tsp"}>tsp</option>
+                          <option value={"tbsp"}>tbsp</option>
+                        </Select>
+                      </Grid>
+                      <Grid item xs={8}>
+                        <TextField
+                          required
+                          fullWidth
+                          key={idx}
+                          name={ingId}
+                          label="Ingredient"
+                          id={ingId}
+                          alt="ingredient"
+                          value={this.props.ingredients[idx].ingredient}
+                          onChange={() => {}}
+                          inputProps={{
+                            "data-id": idx,
+                            "data-field-type": "ingredient"
+                          }}
+                        />
+                      </Grid>
+                    </Grid>
+                  );
+                })}
+                <Grid item xs={12}>
+                  <Button
+                    variant="contained"
+                    color="primary"
+                    startIcon={<FiPlusCircle />}
+                    onClick={this.addIngredient}
+                  >
+                    Add Ingredient
+                  </Button>
+                </Grid>
+              </Grid>
+            </Paper>
+            {/* Preparation Title */}
+            <Grid item xs={12}>
+              <Typography variant="h6" id="difficulty-slider">
+                Preparation
+              </Typography>
+            </Grid>
+            {/* Preparation */}
```

Comment:
Das Feld ist required, aber es wäre schön, wenn es noch einen `*` beim Label gäbe

Reply:
Ja, schade, dass es das nicht per default macht. 
So würde es gehardcoded aussehen: 
![firefox_l07V6LejXu](https://user-images.githubusercontent.com/35599391/74091691-62ca0280-4aba-11ea-968d-6ac9febfdefa.png)
Geht in Ordnung oder?